From: Rob Browning <rlb@defaultvalue.org>
Date: Sun, 5 Feb 2023 07:04:15 +0900
Subject: look-for-news-to-find-etc

* Emacs will now look for NEWS in order to find etc/ rather than GNU.
  Patch: look-for-news-to-find-etc.diff
  Provided-by: Sven Joachim <svenjoac@gmx.de>
  Originally-reported-by: Bernhard Michler <Boregard@gmx.net>
  Date: Mon, 28 Apr 2008 11:20:23 +0200
  Added-by: Rob Browning <rlb@defaultvalue.org>
  Status: Debian specific

  Since Debian doesn't include GNU in the Debian packages (the common
  licenses are in /usr/share/common-licenses), Emacs will now look for
  NEWS to find the etc directory instead of GNU.
---
 src/callproc.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/callproc.c b/src/callproc.c
index 541974f..0f7b19c 100644
--- a/src/callproc.c
+++ b/src/callproc.c
@@ -1587,14 +1587,14 @@ init_callproc ()
 
       srcdir = Fexpand_file_name (build_string ("../src/"),
 				  build_string (PATH_DUMPLOADSEARCH));
-      tem = Fexpand_file_name (build_string ("GNU"), Vdata_directory);
+      tem = Fexpand_file_name (build_string ("NEWS"), Vdata_directory);
       tem1 = Ffile_exists_p (tem);
       if (!NILP (Fequal (srcdir, Vinvocation_directory)) || NILP (tem1))
 	{
 	  Lisp_Object newdir;
 	  newdir = Fexpand_file_name (build_string ("../etc/"),
 				      build_string (PATH_DUMPLOADSEARCH));
-	  tem = Fexpand_file_name (build_string ("GNU"), newdir);
+	  tem = Fexpand_file_name (build_string ("NEWS"), newdir);
 	  tem1 = Ffile_exists_p (tem);
 	  if (!NILP (tem1))
 	    Vdata_directory = newdir;
